pkg/kor/formatter.go

```diff
@@ -0,0 +1,209 @@
+package kor
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/olekukonko/tablewriter"
+)
+
+type ResourceInfo struct {
+	Name   string `json:"name"`
+	Reason string `json:"reason,omitempty"`
+}
+
+func unusedResourceFormatter(outputFormat string, outputBuffer bytes.Buffer, opts Opts, jsonResponse []byte) (string, error) {
+	switch outputFormat {
+	case "table":
+		return outputBuffer.String(), nil
+	case "json", "yaml":
```

Comment:
YAML output is formatted as JSON
![alt text](image.png)

The following is missing:
```
yamlResponse, err := yaml.JSONToYAML(jsonResponse)
```